example/tests.py

```diff
@@ -182,22 +189,6 @@ def test_add_edit_delete_switch(self):
                                                           wait_time=10)
         assert_true(is_deleted, 'Switch was not deleted.')
 
-    def test_switch_history(self):
```

Comment:
So I guess, as part of this simplification, you are removing a switch history feature?
Was that used/working before?